### PR TITLE
runtime: a few more changes to the runtime files

### PR DESCRIPTION
**Pull request by chrisbra**
as mentioned here: https://github.com/vim/vim/commit/1688938dd5ac78ab67e54299b9d5b93499dba762#r124379828

### CODE REVIEW

File: runtime/syntax/shared/typescriptcommon.vim [hunk 1]
```diff
@@ -1,9 +1,9 @@
 " Vim syntax file
 " Language:     TypeScript and TypeScriptReact
-" Maintainer:   Bram Moolenaar, Herrington Darkholme
-" Last Change:	2021 Sep 22
+" Maintainer:   Moolenaar, Herrington Darkholme
```
Comment (ZuBB) [inline runtime/syntax/shared/typescriptcommon.vim]: Only the first name was deleted

Comment (chrisbra) [reply runtime/syntax/shared/typescriptcommon.vim]: yes you are right, thanks.

File: runtime/tutor/tutor.vim [hunk 2]
```diff
@@ -1,7 +1,7 @@
 " Vim tutor support file
-" Author: Eduardo F. Amatria <eferna1@platea.pntic.mec.es>
-" Maintainer: Bram Moolenaar
-" Last Change:	2019 Nov 11
+" Author:	Eduardo F. Amatria <eferna1@platea.pntic.mec.es>
+" Maintainer:	The·Vim·Project·<https://github.com/vim/vim>
```
Comment (seandewar) [inline runtime/tutor/tutor.vim:3]:
```suggestion
" Maintainer:	The Vim Project <https://github.com/vim/vim>
```

File: runtime/syntax/zimbu.vim [hunk 3]
```diff
@@ -1,7 +1,8 @@
 " Vim syntax file
 " Language:	Zimbu
-" Maintainer:	Bram Moolenaar
-" Last Change:	2014 Nov 23
+" Maintainer:	The·Vim·Project·<https://github.com/vim/vim>
```
Comment (seandewar) [inline runtime/syntax/zimbu.vim:3]:
```suggestion
" Maintainer:	The Vim Project <https://github.com/vim/vim>
```

**Review by seandewar — COMMENTED** [review]
I think these are supposed to be spaces

**Pull request merged**